_includes/blog/disqus.html

```diff
@@ -0,0 +1,14 @@
+{%- if page.comments != false and jekyll.environment == "production" -%}
```

Comment:
disqus comments are off. any reason this file cant be deelted?

Reply:
way ahead of u lmao